Library/Formula/ircii.rb

```diff
@@ -1,6 +1,6 @@
 class Ircii < Formula
   desc "IRC and ICB client"
-  homepage "http://www.eterna.com.au/ircii/"
+  homepage "https://web.archive.org/web/20230930141048/http://www.eterna.com.au/ircii/"
```

Comment:
http://www.eterna23.net/ircii/

Reply:
Nice - that's better. https://github.com/mistydemeo/tigerbrew/pull/1365/commits/7b546870ec0c49b08996bc3af314d558dcab092f